docs/src/pages/components/rating/RatingPropControls.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {
+  RatingProps,
+  TextField,
+  SelectField,
+  Flex,
+} from '@aws-amplify/ui-react';
+
+export interface RatingPropControlsProps extends RatingProps {
+  setValue: (value: React.SetStateAction<RatingProps['value']>) => void;
+  setMaxValue: (value: React.SetStateAction<RatingProps['maxValue']>) => void;
+  setSize: (value: React.SetStateAction<RatingProps['size']>) => void;
+  setFillColor: (value: React.SetStateAction<RatingProps['fillColor']>) => void;
+  setEmptyColor: (
+    value: React.SetStateAction<RatingProps['emptyColor']>
+  ) => void;
+}
+
+interface RatingPropControlsInterface {
+  (props: RatingPropControlsProps): JSX.Element;
+}
+
+export const RatingPropControls: RatingPropControlsInterface = ({
+  value,
+  maxValue,
+  size,
+  fillColor,
+  emptyColor,
+  setValue,
+  setMaxValue,
+  setSize,
+  setFillColor,
+  setEmptyColor,
+}) => {
+  return (
+    <Flex direction="column">
+      <TextField
+        label="Value"
+        value={value}
+        placeholder="Set Value"
+        onChange={(event) => setValue(event.target.value)}
+      />
+
+      <TextField
+        label="MaxValue"
```

Comment:
Let's match the labels to the exact prop names. For example, `"MaxValue"` => `"maxValue"`, `"Size` => `size"`, etc